### ISSUE TITLE
Some tests fail using latest version of Mojolicious (6.33)

### ISSUE THREAD

**Issue by ziali088** [plicease/Mojolicious-Plugin-TtRenderer#44]
Hello,

Some updates to Mojolicious seem to be breaking the tests for this module. I couldn't pin down what actually caused it, but I think the relevant changes may be here: https://github.com/kraih/mojo/commit/c7a22d766589f0547f2424dcb5cdcc97ebbbe097

The failed tests are:
```
#   Failed test 'content is similar'
#   at t/default_handler_as_engine.t line 34.
#                   ''
#     doesn't match '(?^:foo)'
# Looks like you failed 1 test of 9.
t/default_handler_as_engine.t .........
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/9 subtests

#   Failed test 'content is similar'
#   at t/default_handler_as_plugin.t line 34.
#                   ''
#     doesn't match '(?^:foo)'
# Looks like you failed 1 test of 9.
t/default_handler_as_plugin.t .........
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/9 subtests

#   Failed test 'content is similar'
#   at t/default_handler_no_templates_dir.t line 20.
#                   ''
#     doesn't match '(?^:foo)'
# Looks like you failed 1 test of 6.
t/default_handler_no_templates_dir.t ..
Dubious, test returned 1 (wstat 256, 0x100)
```

**Comment by plicease** [comment]
Thanks for the report.  Your pointer to the change seems promising, I can see how that would probably break ttrenderer.  I will take a look when I get a chance.

**Comment by plicease** [comment]
This should be fixed in 1.57.  Please let me know if you find otherwise.  Thanks again for the report.

**Comment by ziali088** [comment]
I was able to build successfully with version 1.57. Thanks for your quick response, and also for this plugin! :)
